### PR TITLE
Showcase multilingual dense retrieval with E5

### PR DESCRIPTION
**Pull request by joshdevins**
This new notebook shows a simple walkthrough of using the E5 multilingual embedding model to query a mixed language corpus. It follows the same level of abstraction and details as the other notebooks in this directory, opting to perform all inference outside of the stack. We use select passages from the MIRACL dataset in English and German.

### CODE REVIEW

**Comment by leemthompo** [comment]
👋 @joshdevins as part of https://github.com/elastic/elasticsearch-labs/pull/8 we renamed the notebooks folder to simply `notebooks`, if you don't mind targeting that folder!

File: colab-notebooks-examples/search/04-multilingual.ipynb [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+    " The dataset used contains snippets of Wikipedia passages from the [MIRACL](https://project-miracl.github.io/) dataset."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Y01AXpELkygt"
+   },
+   "source": [
```
Comment (joemcelroy) [inline colab-notebooks-examples/search/04-multilingual.ipynb:33]: we can drop these requirements here. We have done for most recent notebook updates + only calling out ML node needs for when model is used in stack

File: colab-notebooks-examples/search/04-multilingual.ipynb [hunk 2]
```diff
@@ -0,0 +1,487 @@
...
+    "    return client.search(index=\"articles\", knn=knn)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pretty_response(query(\"health\"))"
```
Comment (joemcelroy) [inline colab-notebooks-examples/search/04-multilingual.ipynb:425]: suggestion: include the std_out for these responses.

File: colab-notebooks-examples/search/04-multilingual.ipynb [hunk 3]
```diff
@@ -0,0 +1,487 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pretty_response(query(\"Mauer\", language=\"de\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
```
Comment (joemcelroy) [inline colab-notebooks-examples/search/04-multilingual.ipynb:457]: typo: empty code frame.

File: colab-notebooks-examples/search/04-multilingual.ipynb [hunk 4]
```diff
@@ -0,0 +1,487 @@
...
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "gEzq2Z1wBs3M"
+   },
+   "source": [
+    "[TODO: Update]\n",
```
Comment (leemthompo) [inline colab-notebooks-examples/search/04-multilingual.ipynb:100]:
```suggestion
    "\n",
```

File: colab-notebooks-examples/search/04-multilingual.ipynb [hunk 5]
```diff
@@ -0,0 +1,487 @@
...
+   "outputs": [],
+   "source": [
+    "pretty_response(query(\"health\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that in the results above, we see that the document about healthcare,\n",
```
Comment (leemthompo) [inline colab-notebooks-examples/search/04-multilingual.ipynb:432]: Might need another German speaker to verify the results 😄

File: colab-notebooks-examples/search/04-multilingual.ipynb (hunk 4 above)
Comment (joshdevins) [reply colab-notebooks-examples/search/04-multilingual.ipynb]: I copied that from another notebook. We might need to clean this up elsewhere as well!

File: colab-notebooks-examples/search/04-multilingual.ipynb (hunk 5 above)
Comment (joshdevins) [reply colab-notebooks-examples/search/04-multilingual.ipynb]: I am a German speaker 😉

**Comment by joshdevins** [comment]
@joemcelroy @leemthompo I've rebased and should have all the new stuff from last week. I've also addressed all of the comments from the older version. I made sure to update READMEs and tried to normalise as much as I could see between all the notebooks in `search`. Please have another pass and let me know if this looks good! I'm hoping to complete a blog post this week before PTO and I'll be linking to this notebook.

File: notebooks/search/01-keyword-querying-filtering.ipynb [hunk 6]
```diff
@@ -1,14 +1,5 @@
...
   "cells": [
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "0wgbLWl2udLQ"
-      },
-      "source": [
-        "[Quick Start](https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/search/00-quick-start.ipynb) || **Keyword Querying Filtering** || [Hybrid search with RRF](https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/search/02-hybrid-search.ipynb) || [ELSER](https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/search/03-ELSER.ipynb)"
-      ]
-    },
```
Comment (leemthompo) [inline notebooks/search/01-keyword-querying-filtering.ipynb:11]: @joshdevins is just a drive-by cleanup? :)

**Review by leemthompo — APPROVED** [review]
LGTM, consider adding a blurb in the `search` notebooks [README](https://github.com/elastic/elasticsearch-labs/tree/main/notebooks/search), probably just a condensed bulleted version of your blog intro will do the trick. 

Just started adding those category readmes incrementally this week. :)

**Review by joemcelroy — APPROVED** [review]

File: notebooks/search/01-keyword-querying-filtering.ipynb (hunk 6 above)
Comment (joshdevins) [reply notebooks/search/01-keyword-querying-filtering.ipynb:11]: Yeah, I wanted to normalise all the notebooks and this one stood out as having a header that none of the others had.
